coalib/parsing/DefaultArgParser.py

```diff
@@ -208,6 +208,11 @@ def default_arg_parser(formatter_class=None):
         help='apply all patches automatically if possible')
 
     misc_group.add_argument(
+        '-g', '--ignore-result', action='store_const',
+        dest='default_actions', const='*: IgnoreResultAction',
+        help='ignore a result')
```

Comment:
More explanation needed. I didn't realise it's going to write to the files from this description. I thought it would just ignore the result for this run